src/Numeric/Interval/Internal.hs

```diff
@@ -79,6 +82,14 @@ data Interval a = I !a !a | Empty deriving
 #endif
   )
 
+-- | '<>' is 'hull'
```

Comment:
Since `Data.Semigroup` is imported qualified, I think this needs to be written as `'Data.Semigroup.<>'` in order to be rendered properly. (Check me on that.) Similarly for the other two `Semigroup` instances added in this PR.

Reply:
Good point. It actually worked as `<>` is in GHC-8.6 prelude, but changed to `Data.Semigroup.<>` so it works if someone uses haddock with older GHCs.